state/protocol/badger/mutator.go

```diff
@@ -293,22 +269,15 @@ func (m *ParticipantState) Extend(ctx context.Context, candidateProposal *flow.P
 		return err
 	}
 
-<<<<<<< HEAD
-	// The following function rejects the input block with an [state.OutdatedExtensionError] if and only if
-	// the block is orphaned or already finalized. If the block was to be finalized already, it would have been
-	// detected as already processed by the check above. Hence, `candidate` being orphaned is the only
-	// possible case to receive an [state.OutdatedExtensionError] here.
-	err = m.checkOutdatedExtension(candidate.Header)
-=======
-	// check if the block header is a valid extension of parent block
-	err = m.headerExtend(ctx, candidateProposal, nil, deferredDbOps)
```

Comment:
We have removed the `headerExtend` from here but the statement for error handling:
```
	if err != nil {
		return fmt.Errorf("header not compliant with chain state: %w", err)
	}
```
is still there. I would suggest to keep the old ordering of functions, please refer to this comment first: https://github.com/onflow/flow-go/pull/7795/files#r2315311140

Reply:
Fixed [1a6d796](https://github.com/onflow/flow-go/pull/7795/commits/1a6d79632ec99db3d4b316b4b45cc26e855a1e5d)_